contrib/systemd/prometheus-mumble-exporter.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Prometheus blackbox exporter for Mumble metrics
+After=network.target nss-lookup.target
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/conf.d/prometheus-mumble-exporter
+ExecStart=/usr/bin/mumble-exporter $MUMBLE_EXPORTER_ARGS
+User=mumble-exporter
+Group=mumble-exporter
```

Comment:
Using DynamicUser is more useful than creating a new user for the exporter IMHO. See #1 and https://www.freedesktop.org/software/systemd/man/systemd.exec.html#DynamicUser=.

Reply:
yes. thx for suggestion [48508d8](https://github.com/mguentner/mumble_exporter/pull/2/commits/48508d876710ee3cbab78f6fea8399cfefc9869b)